contentcuration/contentcuration/static/js/edit_channel/channel_set/util.js

```diff
@@ -0,0 +1,56 @@
+var Models = require('../models');
+const State = require("edit_channel/state");
+
+exports.loadChannelSetChannels = function(token) {
```

Comment:
For named exports, the way you have done it above in the constants file is more idiomatic.

Can just do: `export function loadChannelSetChannels function(token) {` instead of what you have here.